src/bin/common.rs

```diff
@@ -224,12 +225,30 @@ pub fn parse_cli() -> CliOptions {
           "no_scene_detection",
         ])
     )
-    .get_matches();
+    .subcommand(SubCommand::with_name("advanced")
```

Comment:
It might make sense to hide this from the CLI help text.
```suggestion
    .subcommand(SubCommand::with_name("advanced")
                .setting(AppSettings::Hidden)
```

Reply:
+1 on hiding it

Reply:
So be it, I wonder if there is a mean to add `--long-help` or such.